hpx/runtime/components/server/runtime_support.hpp

```diff
@@ -160,10 +160,10 @@ namespace hpx { namespace components { namespace server
 
         /// \brief Actions to create new objects
         template <typename Component>
-        HPX_HOST_DEVICE naming::gid_type create_component();
+        naming::gid_type create_component();
 
```

Comment:
I would like to avoid breaking NVCC builds just to fix the clang builds. Let's try to keep NVCC in working state as long as possible...